Client/Teztez/Core/AppDelegate/AppDelegate.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  AppDelegate.swift
+//  Teztez
+//
+//  Created by Almas Zainoldin on 4/18/20.
+//  Copyright © 2020 crococoders. All rights reserved.
+//
+
+import UIKit
+
+@UIApplicationMain
+class AppDelegate: UIResponder, UIApplicationDelegate {
```

Comment:
Due to the fact that we decided to set deployment target as **13.0** - `AppDelegate` changes to `SceneDelegate`, so we need to adjust Coordinators setup accordingly.

- https://aplus.rs/2020/coordinator-pattern-for-ios13/
- https://markstruzinski.com/2019/08/using-coordinator-with-scene-delegates/